client/console_client.go

```diff
@@ -105,15 +153,17 @@ func Make(apiParameter ApiParameter) (*Client, error) {
 
 func MakeFromEnv() (*Client, error) {
 	apiParameter := ApiParameter{
-		BaseUrl:     os.Getenv("CDK_BASE_URL"),
-		Debug:       utils.CdkDebug(),
-		Key:         os.Getenv("CDK_KEY"),
-		Cert:        os.Getenv("CDK_CERT"),
-		Cacert:      os.Getenv("CDK_CACERT"),
-		ApiKey:      os.Getenv("CDK_API_KEY"),
-		CdkUser:     os.Getenv("CDK_USER"),
-		CdkPassword: os.Getenv("CDK_PASSWORD"),
-		Insecure:    strings.ToLower(os.Getenv("CDK_INSECURE")) == "true",
+		BaseUrl:          os.Getenv("CDK_BASE_URL"),
+		Debug:            utils.CdkDebug(),
+		Key:              os.Getenv("CDK_KEY"),
+		Cert:             os.Getenv("CDK_CERT"),
+		Cacert:           os.Getenv("CDK_CACERT"),
+		ApiKey:           os.Getenv("CDK_API_KEY"),
+		CdkUser:          os.Getenv("CDK_USER"),
+		CdkPassword:      os.Getenv("CDK_PASSWORD"),
+		ExternalUser:     os.Getenv("CDK_EXTERNAL_USER"),
```

Comment:
If you have a better idea for the name of the env var, I'm all ears.

Reply:
REMOTE_AUTH_USER/REMOTE_AUTH_PASS ?

Reply:
CDK_JWT_OAUTH_CLIENT_ID / CDK_JWT_OAUTH_CLIENT_SECRET ?
Or reuse CDK_USER/CDK_PASSWORD and add a CDK_AUTH_MODE with `LOCAL`, `APIKEY`, `EXTERNAL_BASIC` 
because this new user and password are used into a basic authentication header not supported by conduktor directly. 
As we might want  to unify the console and gateway configuration like we do in the terraform provider add new config will just make it more complicated no ?

Reply:
And here we want to support a very specific usecase with an api-gateway in front of console that do the OAUTH workflow from a client ID and secrets passed as basic auth to forward a queries with IDP JWT instead of Console own authentication mechanisms.

Reply:
If we want to emphasize the use case, we could have CDK_IDP_USER/CDK_IDP_PASSWORD ?
Reusing USER/PASSWORD could be interesting I'm afraid that we will break current setups if we have to set `CDK_AUTH_MODE=local` to use conduktor login with user/password.

Reply:
CDK_AUTH_MODE=local could be the default

Reply:
CDK_AUTH_MODE=apikey is usually the default
or we should have only 2 auth modes:
- CONDUKTOR (default)
- EXTERNAL_IDP or API_GATEWAY